momentum-design/tokens
# feat(docs): add tokens to docs

# Description

This pr adds Docosaurus docs and scripts to generate web tokens into docs pages.
To test the docs page, follow these steps:

0. Run `yarn build:web:docs` to make sure you generate docs for the latest tokens  
1. `cd docs && npm i`
2. `npm run start`
3. Open `localhost:3000`

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/14194394/159746951-3256df4b-e637-421d-930f-945ea0088775.png">
